rules/prefer-query-selector.js

```diff
@@ -11,17 +11,37 @@ const disallowedIdentifierNames = new Map([
 	['getElementById', 'querySelector'],
 	['getElementsByClassName', 'querySelectorAll'],
 	['getElementsByTagName', 'querySelectorAll'],
+	['getElementsByName', 'querySelectorAll'],
 ]);
 
 const getReplacementForId = value => `#${value}`;
 const getReplacementForClass = value => value.match(/\S+/g).map(className => `.${className}`).join('');
+const getReplacementForName = (value, originQuote) => value.match(/\S+/g).map(name => `[name=${wrapQuoted(name, originQuote)}]`).join('');
```

Comment:
```suggestion
const getReplacementForName = (value, originalQuote) => `[name=${wrapQuoted(value, originalQuote)}]`;
```

I don't think `getElementsByName` accepts mutiple names,

Reply:
You are right, and I have added the test for multiple names.